.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
```

Comment:
Decided to leave blank issues as an option, for now. I don't have a strong feeling about this, so I'm okay to change it if anyone wants it to happen.